Lib/platform.py

```diff
@@ -1190,12 +1189,19 @@ def _sys_version(sys_version=None):
 
     else:
         # CPython
-        match = sys_version_parser.match(sys_version)
+        cpython_sys_version_parser = re.compile(
+            r'([\w.+]+)\s*'  # "version<space>"
+            r'(free-threading)?\s*' # "free-threading<space>"
```

Comment:
`(?:` would make this non-capturing so you could get rid of the additional `, _,` in the match assignment below?

Reply:
Like this:

```suggestion
            r'(?:free-threading)?\s*' # "free-threading<space>"
```